common/static/common/js/discussion/views/discussion_thread_list_view.js

```diff
@@ -320,10 +321,13 @@
                 var threadCommentCount = thread.get('comments_count'),
                     threadUnreadCommentCount = thread.get('unread_comments_count'),
                     neverRead = !thread.get('read') && threadUnreadCommentCount === threadCommentCount,
+                    shortenBody = DiscussionUtil.abbreviateString(thread.get('body'), 140),
```

Comment:
Rather than abruptly truncating at 140 characters, have we looked at the possibility of:
- showing just enough characters to fill the available space
- showing a constant number of characters that will always be greater than the amount that can fit in the space, and trimming/adding an ellipsis to them with CSS

I don't know what the AC of this story are but it seems like there's more user-friendly options than just a straight truncation.